src/main/kotlin/org/flaxo/plagiarism/model/GraphLink.kt

```diff
@@ -29,5 +29,10 @@ class GraphLink(
          *
          * *The higher weight the closer nodes are.*
          */
-        val weight: Int
+        val weight: Int,
+
+        /**
+         * Direction of the link
+         */
+        val directedTo: Direction
```

Comment:
We should make direction optional because not all graphs may have such an information.

In order to do so:
1. The field should be made nullable.
2. `Optional` annotation from `kotlinx.serialization` should be attached to the field.

Reply:
Now this property is optional and nullable